documentation/certificates.md

```diff
@@ -0,0 +1,91 @@
+Streisand PKI
+=============
+
+Streisand includes a role responsible for PKI certificate generation,
+separated into two distinct tasks:
+
+ - CA/Server
+ - Client(s)
+
+The first of which will generate a certificate authority, and a server 
+certificate signed by the newly minted certificate authority.
+
+The latter task is responsible for generating client side certificate
+(using the same CA) used for authenticting connecting VPN clients.
+
+Each VPN service that leverages PKI generates its own CA/Server and 
+client certificates.
+
+Infrastructure
+--------------
+
+The overall PKI infrasture assumes the following layout:
```

Comment:
typo: "infrasture"

Reply:
I think "layout" might be better described as "structure". What do you think?

Reply:
yup sounds better